crates/autopilot/src/domain/settlement/trade/math.rs

```diff
@@ -6,20 +6,22 @@ use {
             auction::{self, order},
             eth,
             fee,
-            settlement,
+            settlement::{
+                transaction::{ClearingPrices, Prices},
+                {self},
+            },
         },
         util::conv::U256Ext,
     },
     bigdecimal::Zero,
     num::{CheckedAdd, CheckedDiv, CheckedMul, CheckedSub},
 };
 
-/// A single trade executed on-chain, as part of the [`settlement::Solution`].
-///
-/// Referenced as [`settlement::solution::Trade`] in the codebase.
+/// A trade containing bare minimum of onchain information required to calculate
+/// the surplus, fees and score.
 #[derive(Debug, Clone)]
 pub struct Trade {
```

Comment:
nit: can we restrict visibility to super so that we don't leak this type into the rest of the domain